Okta/OktaTokenManager.swift

```diff
@@ -101,3 +101,33 @@ open class OktaTokenManager: NSObject, NSCoding {
         OktaAuth.tokens = nil
     }
 }
+
+extension OktaTokenManager {
+    
+    static let secureStorageKey = "OktaAuthStateTokenManager"
+
+    class func readFromSecureStorage() -> OktaTokenManager? {
+        guard let encodedAuthState: Data = try? OktaKeychain.get(key: secureStorageKey) else {
+            return nil
+        }
+
+        guard let state = NSKeyedUnarchiver.unarchiveObject(with: encodedAuthState) as? OktaTokenManager else {
+            return nil
+        }
+
+        return state
+    }
+    
+    func writeToSecureStorage() {
+        let authStateData = NSKeyedArchiver.archivedData(withRootObject: self)
+        do {
+            try OktaKeychain.set(
+                key: OktaTokenManager.secureStorageKey,
+                data: authStateData,
+                accessibility: self.accessibility
+            )
+        } catch let error {
+            print("Error: \(error)")
```

Comment:
We should avoid using "print" in the libraries. The solution could be:
#if DEBUG
    print(y)
#endif

Or implement Logger class that prints statements in debug mode only
Or logging capability should be part of SDK's config

Reply:
Since this is not the only case when print is used within the SDK, let's create a separate issue and fix it uniformly for the all codebase.

@IldarAbdullin-okta It seams that this fix is related to issue #121. Should we merge it to master or dev?

Reply:
Agree that there should be separate task for logging

Yes, bug fixes that are found in public release should be merged into master branch. Please cherry-pick to dev branch as well